ios/RCTMapboxGL/RCTMapboxGLManager.m

```diff
@@ -133,6 +133,16 @@ - (NSDictionary *)constantsToExport
     }
 }
 
+RCT_CUSTOM_VIEW_PROPERTY(contentInset, UIEdgeInsetsMake, RCTMapboxGL)
+{
+    int top = [json[0] intValue];
```

Comment:
If you declare these variables as `CGFloat`s and set them using `doubleValue` instead of `intValue`, the casts below go away.